functions/Find-DbaDbUnusedIndex.ps1

```diff
@@ -72,11 +96,40 @@ function Find-DbaDbUnusedIndex {
         [switch]$IgnoreUptime,
         [Parameter(ValueFromPipeline)]
         [Microsoft.SqlServer.Management.Smo.Database[]]$InputObject,
-        [switch]$EnableException
+        [switch]$EnableException,
+        [ValidateRange(1, 1000000)][Int]$UserSeeksLessThan = 1,
+        [ValidateRange(1, 1000000)][Int]$UserScansLessThan = 1,
+        [ValidateRange(1, 1000000)][Int]$UserLookupsLessThan = 1
```

Comment:
Once we finalize the parameter name, pleas add examples showing their use.

Reply:
`[int]` instead of `[Int]` as well please 😉

Reply:
ok, sounds good